### PR TITLE
Add project vertical navbar

### PR DESCRIPTION
**Pull request by sofiyareverse**

### CODE REVIEW

**Review by Alexey1100 — CHANGES_REQUESTED** [review]
Visibility of project nav tabs should be based on policies, project settings and token type.
Pls see current implementation of `ProjectSetupHeader.jsx` for the references.

General notes are:
- All of the links should be rendered based on according project policy
- Transfer Rules are only visible for projects using security token
- Tasks page availability should be based on whitelabel settings 
- Project overview shouldn't be available on whitelabel

P.s: should we include external project links, like wiki and GitHub? cc @aquabu

File: app/views/shared/_project_new_styling.html.erb [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                  </svg>
+                </span>
+              </span>
+              <span class="nav-link-title">
+                Overview
+              </span>
+            </a>
+          </li>
+        <% end %>
+        <% if @whitelabel_mission&.project_awards_visible? || !@whitelabel_mission %>
```
Comment (Alexey1100) [inline app/views/shared/_project_new_styling.html.erb:45]: Should be:

```ruby
policy(@project).show_award_types? && (@whitelabel_mission&.project_awards_visible? || !@whitelabel_mission)
```

File: app/views/shared/_project_new_styling.html.erb [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                  </svg>
+                </span>
+              </span>
+              <span class="nav-link-title">
+                Tasks
+              </span>
+            </a>
+          </li>
+        <% end %>
+        <li class="nav-item <%= active_class(project_dashboard_transfers_path(@project.id)) %>">
```
Comment (Alexey1100) [inline app/views/shared/_project_new_styling.html.erb:60]: Should check for:

```ruby
policy(@project).show_contributions?
```

File: app/views/shared/_project_new_styling.html.erb [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                </svg>
+              </span>
+            </span>
+            <span class="nav-link-title">
+              Transfers
+            </span>
+          </a>
+        </li>
+        <li class="nav-item <%= active_class(project_dashboard_accounts_path(@project.id)) %>">
+          <a class="nav-link" href="/projects/<%= @project.id %>/dashboard/accounts">
```
Comment (Alexey1100) [inline app/views/shared/_project_new_styling.html.erb:74]: Should check for:

```ruby
policy(@project).show_contributions?
```

File: app/views/shared/_project_new_styling.html.erb [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+                </svg>
+              </span>
+            </span>
+            <span class="nav-link-title">
+              Accounts
+            </span>
+          </a>
+        </li>
+        <% if current_account&.owned_project?(@project) || !@project.require_confidentiality %>
```
Comment (Alexey1100) [inline app/views/shared/_project_new_styling.html.erb:86]: Should use policy:

```ruby
policy(@project).show_transfer_rules?
```

File: app/views/shared/_project_new_styling.html.erb [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+                  </svg>
+                </span>
+              </span>
+              <span class="nav-link-title">
+                Transfer Rules
+              </span>
+            </a>
+          </li>
+        <% end %>
+        <% if current_account&.owned_project?(@project) %>
```
Comment (Alexey1100) [inline app/views/shared/_project_new_styling.html.erb:104]: Should use policy:

```ruby
policy(@project).accesses?
```

File: app/views/shared/_project_new_styling.html.erb [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+              <span class="nav-link-icon-active">
+                <span class="nav-link-icon d-lg-inline-block"><svg xmlns="http://www.w3.org/2000/svg" class="icon" width="16" height="16" viewBox="0 0 21 21" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round"><path stroke="none" d="M0 0h24v24H0z" fill="none" /><path d="M4 8v-2a2 2 0 0 1 2 -2h2" /><path d="M4 16v2a2 2 0 0 0 2 2h2" /><path d="M16 4h2a2 2 0 0 1 2 2v2" /><path d="M16 20h2a2 2 0 0 0 2 -2v-2" /><rect x="8" y="11" width="8" height="5" rx="1" /><path d="M10 11v-2a2 2 0 1 1 4 0v2" /></svg>
+                </span>
+              </span>
+              <span class="nav-link-title">
+                Access
+              </span>
+            </a>
+          </li>
+          <li class="nav-item <%= active_class(edit_project_path(@project.id)) %>">
```
Comment (Alexey1100) [inline app/views/shared/_project_new_styling.html.erb:116]: Should use policy:

```ruby
policy(@project).edit?
```

**Review by Alexey1100 — CHANGES_REQUESTED** [review]
Pls use according policy for each nav element.

**Review by Alexey1100 — CHANGES_REQUESTED** [review]
@sofiyareverse great job!

The icons and fonts looks a bit larger than they should, could you fix that?
Also non-collapsed view shouldn't include icons.


https://user-images.githubusercontent.com/10757562/127689824-d8a5b33a-01d2-4904-a06d-f1cd82e97c0c.mov

**Comment by sofiyareverse** [comment]
> @sofiyareverse great job!
> 
> The icons and fonts looks a bit larger than they should, could you fix that?
> Also non-collapsed view shouldn't include icons.
> 
>  Screen.Recording.2021-07-30.at.20.22.27.mov

I could but if you look to figma pictures you 'll that there are large pictures for the project nav

**Review by zhinfuhnya — APPROVED** [review]

**Review by Alexey1100 — APPROVED** [review]

**Pull request merged**